src/meta/api/src/schema_api_impl.rs

```diff
@@ -2433,6 +2433,27 @@ impl<KV: kvapi::KVApi<Error = MetaError> + ?Sized> SchemaApi for KV {
         Ok(db_names)
     }
 
+    #[logcall::logcall("debug")]
+    #[minitrace::trace]
+    async fn get_table_name_by_id(&self, table_id: MetaId) -> Result<String, KVAppError> {
```

Comment:
There is no `tenant` parameter passed in? This might conflict with our ongoing tenant level data export feature.

cc @drmingdrmer

Reply:
That's OK for now. I'll refactoring this part in future, when `TableId` is updated to be with a `Tenant` in it.